node.js

```diff
@@ -258,5 +272,33 @@ module.exports = {
   clearCaches: function clearCaches () {
     filenessCache = { }
     configCache = { }
+  },
+
+  checkCanIUse: function checkCanIUse (agentsObj) {
+    var newestBrowserReleaseTime = getNewestReleaseTime(agentsObj)
+    var today = new Date()
+    var halfYearAgo = new Date(
+      today.getFullYear(),
+      today.getMonth() - MONTH_TO_UPDATE_CANIUSE,
+      today.getDate()
```

Comment:
It could generate invalid date if today in 31 days of month and `month - 6` will have only 28/30 days.

It is better just use `Date.now() - 6 * 30 * 24 * 60 * 60 * 1000`